requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy==1.19.2
+matplotlib==3.3.4
+scikit_learn==0.24.1
+opencv==4.5.1

```

Comment:
Please include os, shutil also as they are mentioned as dependencies in README.

Reply:
Actually, it's inbuilt in python I believe(cause I tried on my PC).
I just have python installed and the ```import shutil``` works fine.
can you please confirm?
Same for os
pip install requirements.txt

Reply:
yes, if we write import statement at the top of program, it works fine as expected. 
Purpose of having requirements file was though we don't mention required packages inside program it should work fine.
Hence, please mention in requirements file too.

Reply:
Okay !! I'll do it now